reactive-streams/src/main/scala/fs2/interop/reactivestreams/StreamSubscriber.scala

```diff
@@ -69,15 +69,19 @@ final class StreamSubscriber[F[_], A](
     sub.onError(t)
   }
 
-  def stream(subscribe: F[Unit]): Stream[F, A] = sub.stream(subscribe)
+  def streamChunk(subscribe: F[Unit]): Stream[F, Chunk[A]] = sub.stream(subscribe)
+  def stream(subscribe: F[Unit]): Stream[F, A] = streamChunk(subscribe).unchunks
```

Comment:
Echoing @diesalbla's comment here -- I think we should offer just the `stream(...): Stream[F, A]` method and let folks manually call `.chunks` if they want to operate on chunks explicitly.

Reply:
https://github.com/typelevel/fs2/pull/2666/commits/d99c684026159103000af81b482f4c6de25fc6c3